keys.json

```diff
@@ -1,82 +1,32 @@
-    "coinegg":       { "skip": true },
-    "coinex":       { "skip": true },
-    "coinfalcon":    { "skip": true },
-    "coingi":        { "skip": true },
-    "coinbase":      { "skip": true },
-    "coinmarketcap": { "skip": true },
-    "coinmate":      { "skip": true },
-    "coinone":       { "skip": true },
-    "currencycom":   { "skip": true, "skipWs": true },
-    "digifinex":     { "skip": true },
-    "dsx":           { "skip": true, "skipWs": true },
-    "equos":         { "skip": true },
-    "eterbase":      { "skip": true },
-    "ethfinex":      { "skip": true },
-    "exx":           { "skip": true },
-    "fcoin":         { "skip": true },
-    "fcoinjp":       { "skip": true },
-    "flowbtc":       { "skip": true },
-    "gemini":        { "skip": true },
-    "huobipro":      { "skip": true, "skipWs": true },
-    "itbit":         { "skip": true },
-    "indodax":       { "skip": true },
-    "kuna":          { "skip": true },
-    "latoken2":      { "skip": true },
-    "lbank":         { "skip": true },
-    "mixcoins":      { "skip": true },
-    "mexc":          { "skip": true },
-    "mexc3":         { "skip": true },
-    "okcoin":        { "skip": true, "skipWs":true },
-    "poloniex":      { "skip": true, "skipWs": true },
-    "qtrade":        { "skip": true },
-    "rightbtc":      { "skip": true },
-    "stex":          { "skip": true },
-    "tokocrypto":    { "skip": true },
-    "tidebit":       { "skip": true },
-    "tidex":         { "skip": true },
-    "vcc":           { "skip": true },
-    "woo":           { "skip": true },
-    "yobit":         { "skip": true },
-    "zaif":          { "skip": true },
-    "zipmex":        { "skip": true, "skipWs": true  },
+    "bitfinex":      { "skipWs": true },
+    "cdax":          { "skipWs": true },
+    "currencycom":   { "skipWs": true },
+    "dsx":           { "skipWs": true },
+    "huobipro":      { "skipWs": true },
+    "okcoin":        { "skipWs": true },
+    "poloniex":      { "skipWs": true },
+    "zipmex":        { "skipWs": true },
     "gate":          { "skipWs": true },
     "gateio":        { "skipWs": true },
     "bequant":       { "skipWs": true },
     "binancecoinm":  { "skipWs": true },
     "bitopro":       { "skipWs": true },
     "bitvavo":       { "skipWs": true },
-    "ftx":           { "skipWs":  true },
-    "huobi":         { "skipWs":  true },
+    "ftx":           { "skipWs": true },
+    "huobi":         { "skipWs": true },
     "hollaex":       { "skipWs": true },
     "idex":          { "skipWs": true },
     "kucoin":        { "skipWs": true },
     "ndax":          { "skipWs": true },
     "okex":          { "skipWs": true },
     "ripio":         { "skipWs": true },
     "upbit":         { "skipWs": true },
-    "zb":            { "skipWs": true }
+    "zb":            { "skipWs": true },
+    "tidebit":       { "skip": true },
+    "tokocrypto":    { "skip": true }
```

Comment:
I've not removed these two exchanges from skipped rest tests, as they still had some issues, that needed fix, but wasn't suitable for this PR, so i've left them. they'll be fixed separately in near future.